libs/partners/milvus/langchain_milvus/vectorstores/milvus.py

```diff
@@ -814,7 +839,7 @@ def add_texts(
 
     def _collection_search(
         self,
-        embedding: List[float],
+        embedding: List[float] | Dict[int, float],
```

Comment:
wil be appreciate if change the corresponding Args docstr below

Reply:
done